Gemfile

```diff
@@ -40,18 +40,18 @@ gem 'rubytree'
 gem 'statsd-ruby'
 gem 'uglifier', '>= 1.3.0'
 
-gem 'advice_plans', '~> 3.1.0'
+gem 'advice_plans', git: 'git@github.com:moneyadviceservice/advice_plans.git', branch: 'unpin_sass_rails'
```

Comment:
You can golf these down slightly with `github: 'moneyadviceservice/repo' #...` thus cutting out the boilerplate git stuff.